### PR TITLE
Add new game Zwanzigerrufen

### LINKED ISSUE #992: Inconsistent card face when toggling between Pattern Types
**Describe the bug**
When card is opened in storybook and the controls are toggled between Tarot -> Franconian the card is left blank; when the card is toggled between Skat -> Franconian the card face is misaligned. I suppose one could argue this use case is out-of-scope -- but we're the common code so we don't know all the described uses.

UPDATE: This behavior appears to be isolated to "Canvas" and not "Docs" view.

**To Reproduce**
1. Go to 'docs:games shared : components: cards : card'
2. Click on the tab for 'canvas'.
3. Click on 'controls tab and toggle the Pattern radio buttons'
4. When moving between Tarot -> Franconian the card face is empty
5. When moving between Skat -> Franconian the card face is misaligned with the master sheet of cards.
6. If you refresh the browser while on Franconian the face is populated and will work correctly until a refresh on Tarot.
7. If you use the Docs tab this behavior is absent (makes me believe it's a sb defect)

**Expected behavior**
Moving between these types should always yield a valid face

**Video**
[Demonstration of defect using storybook](https://vimeo.com/638002214/ad304ad7e5)
[Demo on Chrome, Safari, MacOS Big Sur](https://vimeo.com/638157253/213528b823)
**Desktop (please complete the following information):**
 - OS: [MacOS Big Sur]
 - Browser [Safari 15.0, Chrome 94.0.4606.81]
 - Version [] **<--- is this supposed to be game version?**

**Additional context**
Really suspecting this is a regression on sb@6.0.22 where I don't see the behavior in PRD.

Now present in PRD since PR #991

### PR DESCRIPTION
**Pull request by tuxor1337**
This adds an implementation of the eastern Austrian game [Zwanzigerrufen](https://en.wikipedia.org/wiki/Zwanzigerrufen) which is a simplified version of [Königrufen](https://en.wikipedia.org/wiki/K%C3%B6nigrufen):

grafik

The new card pattern is a traditional Tarock pattern used in Austria since the early 19th century, hence it is public domain.

While implementing this game, I fixed #992 and some other card game-related UI stuff.

#### Checklist

* [x] Use a separate branch in your local repo (not `master`).
* [x] I've read and agree with the [contribution guidelines](https://www.freeboardgames.org/docs/?path=/story/documentation-contributing--page).

### CODE REVIEW

File: web/src/gamesShared/components/cards/Hand.tsx [hunk 1]
```diff
@@ -21,12 +18,21 @@ export function Hand(props: {
...
-      width: `${scale * CARD_WIDTH_MIN + (1 - scale) * CARD_WIDTH_MAX}px`,
-    };
+    let width_min = 35;
+    let width_max = 75;
+    let visible_ratio = 0.33;
+    if (props.pattern == Pattern.Tarock) {
+      width_min = 60;
+      width_max = 90;
+      visible_ratio = 0.6;
+    }
```
[the remark this answers is not in the document]
Comment (tuxor1337) [reply web/src/gamesShared/components/cards/Hand.tsx:28]: I don't understand your idea: Which alias would you like to have for which existing type?

File: web/src/gamesShared/components/cards/PlayerZones.module.css [hunk 2]
```diff
@@ -4,7 +4,6 @@
   bottom: 0;
   left: 0;
   right: 0;
-  margin: 15px 15px 0;
```
[the remark this answers is not in the document]
Comment (cwatsonc) [reply web/src/gamesShared/components/cards/PlayerZones.module.css:7]: > Maybe let's discuss this in a different context, e.g. in a GitHub issue or on discord.

added [[UI/UX]Tournament Support](https://github.com/freeboardgames/FreeBoardGames.org/issues/996)

File: web/src/games/frenchTarot/board.tsx [hunk 3]
```diff
@@ -221,17 +221,17 @@ export function BgioBoard(props: { G: IG; ctx: Ctx; moves: IGameMoves; playerID:
   }
 
   function renderButtonsBid() {
-    if (playerPhase != Phases.bidding || player.bid == 0) return;
+    if (playerPhase != Phases.bidding || player.bid == Contract.Pass) return;
     const highest_bid = Math.max(...G.players.map((P) => P.bid));
-    const all_bids = [0, 1, 2, 3, 4];
+    const all_bids = [Contract.Pass, Contract.Small, Contract.Guard, Contract.GuardWithout, Contract.GuardAgainst];
```
[the remark this answers is not in the document]
Comment (vdfdev) [reply web/src/games/frenchTarot/board.tsx:226]: great, this is much better

File: web/src/games/zwanzigerrufen/locales/pt.json [hunk 4]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Zwanzigerrufen",
+  "description": "Jogo de Cartas da Áustria Oriental",
+  "descriptionTag": "Jogue Zwanzigerrufen online!",
+  "instructions": {
+    "text": "Para uma descrição do jogo, por favor leia a página da Wikipedia: https://en.wikipedia.org/wiki/Zwanzigerrufen\n\nToda a arte usada nessa implementação é do baralho de cartas \"Industrie und Glück\" da primeira metade do século XIX e disponível no domínio público: https://en.wikipedia.org/wiki/Industrie_und_Gl%C3%BCck"
```
[the remark this answers is not in the document]
Comment (vdfdev) [reply web/src/games/zwanzigerrufen/locales/pt.json:6]: pretty good job :)

**Review by vdfdev — APPROVED** [review]

**Pull request merged**